jaeger_client/codecs.py

```diff
@@ -14,9 +14,6 @@
 
 from __future__ import absolute_import
 
-from future import standard_library
-standard_library.install_aliases()  # noqa
-
 from builtins import object
 from past.builtins import basestring
```

Comment:
You dropped the `futures` package from the requirements but are still referencing it.

I get a stacktrace when trying to run this in a Python 3 environment.

```
  File "/usr/src/app/src/jaeger-client/jaeger_client/tracer.py", line 30, in <module>
    from .codecs import TextCodec, ZipkinCodec, ZipkinSpanFormat, BinaryCodec
  File "/usr/src/app/src/jaeger-client/jaeger_client/codecs.py", line 18, in <module>
    from past.builtins import basestring
ModuleNotFoundError: No module named 'past'
```

Reply:
Thanks for reporting - https://github.com/jaegertracing/jaeger-client-python/issues/133